src/openMVG/features/openCV/AKAZE_openCV_describer.cpp

```diff
@@ -0,0 +1,57 @@
+#include "AKAZE_openCV_describer.hpp"
```

Comment:
We plan to make a global pass to change the headers and use a sentence to say that this file is part of the project, as you have done on your side.